### PR TITLE
Versioning Docs: Getting Started Blue-Green

### PR DESCRIPTION
**Pull request by carlydf**

### CODE REVIEW

**Review by drewhoskins-temporal — COMMENTED** [review]
lgtm so far

File: docs/worker-versioning.md [hunk 1]
```diff
@@ -52,41 +52,113 @@ so each new CaN execution starts on the latest Build ID.
...
+### 6. Commit the Build ID to make it the new default version
+This will atomically delete the partially-ramped rule you added in step 4 and replace it with
+a rule with 100% ramp.
+```shell
+temporal task-queue versioning commit-build-id --task-queue my-tq --build-id $BUILD_ID
+```
+See the `--help` output for more details:
+```shell
+% temporal task-queue versioning commit-build-id --help
+Completes  the rollout of a BuildID and cleanup unnecessary rules possibly
```
Comment (drewhoskins-temporal) [inline docs/worker-versioning.md:132]:
```suggestion
Completes  the rollout of a BuildID and cleans up unnecessary rules possibly
```

File: docs/worker-versioning.md [hunk 2]
```diff
@@ -52,41 +52,113 @@ so each new CaN execution starts on the latest Build ID.
...
+temporal task-queue describe --task-queue my-tq --select-build-id $OLD_BUILD_ID --report-reachability
+```
+
+Note: Set `OLD_BUILD_ID` to your previous default version, or to the empty string if your previous default was unversioned.
+If you omit the `--select-build-id` flag, results for the current default Build ID will be returned.
+
+
+### 8. Decommission the previous versioned worker
+After the reachability status for your old version and task queue is `CLOSED_WORKFLOWS_ONLY` you can safely 
+decommission your old worker(s). See [Build ID Reachability](#build-id-reachability) for more details.
```
Comment (drewhoskins-temporal) [inline docs/worker-versioning.md:160]:
```suggestion
Unless your application can query closed workflows, after the reachability status for your old version and task queue is `CLOSED_WORKFLOWS_ONLY` you can safely decommission your old worker(s). 
See [Build ID Reachability](#build-id-reachability) for more details.
```

File: docs/worker-versioning.md [hunk 3]
```diff
@@ -52,41 +52,113 @@ so each new CaN execution starts on the latest Build ID.
...
+To prevent committing invalid Build IDs, we reject the request if no pollers
+have been seen recently for this Build ID. Use the force option to disable this validation.
+```
+
+### 7. Wait until the old workers are not needed
+Check the reachability of your old version. Now that all new workflow executions are assigned to the new Build ID,
+your old worker (that was running in step 0) will not receive any tasks from new workflow executions. Running workflow
+executions will send their outstanding tasks to workers with the version that the workflow execution is assigned to 
+(barring other instructions via [Redirect Rules](#redirect-rules)), so in most cases, a worker is no longer needed 
+after the reachability status transitions from `REACHABLE` to `CLOSED_WORKFLOWS_ONLY`.
```
Comment (drewhoskins-temporal) [inline docs/worker-versioning.md:148]:
```suggestion
(barring other instructions via [Redirect Rules](#redirect-rules)).  Unless your application performs queries to closed workflows, a worker is no longer needed after the reachability status transitions from `REACHABLE` to `CLOSED_WORKFLOWS_ONLY`.
```

File: docs/worker-versioning.md [hunk 4]
```diff
@@ -52,41 +52,113 @@ so each new CaN execution starts on the latest Build ID.
...
+
+### 6. Commit the Build ID to make it the new default version
+This will atomically delete the partially-ramped rule you added in step 4 and replace it with
+a rule with 100% ramp.
+```shell
+temporal task-queue versioning commit-build-id --task-queue my-tq --build-id $BUILD_ID
+```
+See the `--help` output for more details:
+```shell
+% temporal task-queue versioning commit-build-id --help
```
Comment (drewhoskins-temporal) [inline docs/worker-versioning.md:131]: Looks like missing a ``` after this shell command, and remove the one from below?

**Review by ShahabT — APPROVED** [review]

**Pull request merged**